docs/grpc/javascript.md

```diff
@@ -18,49 +19,57 @@ at least somewhere reachable by your Javascript code).
 The `rpc.proto` file is [located in the `lnrpc` directory of the `lnd`
 sources](https://github.com/lightningnetwork/lnd/blob/master/lnrpc/rpc.proto).
 
-In order for the auto-generated code to compile successfully, you'll need to
-comment out the following line:
-
-```
-//import "google/api/annotations.proto";
```

Comment:
Do we know why this isn't necessary anymore?

Reply:
Yes, the `proto-loader` library just ignores these annotations, even if it cannot find the referenced proto file. I've verified it is no longer needed by running the example code on a fresh system.